batch_job/src/batch_job_amazon.c

```diff
@@ -180,19 +296,59 @@ static struct jx * aws_describe_instance( struct aws_config *c, const char *inst
 Terminate an EC2 instance.  If termination is successfully applied, return true, otherwise return false.
 */
 
-static int aws_terminate_instance( struct aws_config *c, const char *instance_id )
+static int aws_terminate_instance( struct batch_queue * q, struct aws_config *c, const char *instance_id )
 {
 	char *str = string_format("aws ec2 terminate-instances --instance-ids %s --output json",instance_id);
 	struct jx *jresult = json_command(str);
 	if(jresult) {
 		jx_delete(jresult);
 		printf("deleted virtual machine instance %s\n",instance_id);
+		FILE *fp;
```

Comment:
This is a bad pattern to start: Write a separate function called aws_log_message() or something that logs the desired message, so you don't keep repeating the get_option/fopen/fclose pattern